spec/stud/interval_spec.rb

```diff
@@ -1,8 +1,19 @@
 require "stud/interval"
 require "spec_env" # from the top level spec/ directory
 
+module Stud
+  def reset!(target = Thread.current)
```

Comment:
good point. I can expose it. for lumberjack, tcp and udp inputs, none of these are using `interval` so that should not be a problem in this specific case.